### PR TITLE
store: Don't do any seek operation after store closed.

### PR DESCRIPTION
**Pull request by ngaut**

### CODE REVIEW

**Comment by c4pt0r** [comment]
LGTM

**Comment by ngaut** [comment]
@coocood PTAL

File: store/localstore/kv.go [hunk 1]
```diff
@@ -150,9 +153,10 @@ func (s *dbStore) scheduler() {
 			}
 		case <-s.closeCh:
 			closed = true
-			s.wg.Done()
 			// notify seek worker to exit
 			close(seekCh)
+			wgSeekWorkers.Wait()
+			s.wg.Done()
```
Comment (coocood) [inline store/localstore/kv.go]: The scheduler function never return.

Comment (ngaut) [inline store/localstore/kv.go]: It should't, we still need to return error when store closed if user try to commit transaction.

Comment (coocood) [inline store/localstore/kv.go]: Then it will never get blocked, keep on running, waste CPU time.

Comment (coocood) [inline store/localstore/kv.go]: Oh, the closeCh will only receive once.

**Comment by coocood** [comment]
LGTM

**Pull request merged**